runtimed/src/routes.rs

```diff
@@ -104,3 +106,8 @@ async fn get_runtime_instance_attach(
 
     Ok(Sse::new(sse_stream).keep_alive(KeepAlive::default()))
 }
+
+async fn get_environments() -> Result<Json<Vec<JupyterKernelspecDir>>, StatusCode> {
+    let kernelspecs = runtimelib::jupyter::list_kernelspecs().await;
+    Ok(Json(kernelspecs))
```

Comment:
We may want to turn this into something that watches the directories or caches them. For another PR or as we need it though.